src/connector.ts

```diff
@@ -134,4 +141,33 @@ export class Connector {
 
 		addedChanged(unprotectString(this.coreHandler.core.deviceId))
 	}
+
+	private setupReloadDataKoaEndpoint(): void {
+		const KOA_PORT: number = 3007
+		const RUNDOWN_EXTERNAL_ID_SUFFIX: string = '_1'
+
+		this.koaRouter.post(
+			'/reloadData/rundowns/:rundownName',
```

Comment:
Instead of defining the resource on the action, the action should be defined on the resource as `/rundowns/:rundownName/reload-data`.

I have changed `reloadData` to `reload-data` to avoid clash between case-sensitive and case-insensitive systems (https://blog.restcase.com/5-basic-rest-api-design-guidelines/, see spinal-case).

Reply:
Thanks for the read, that helped! 
I changed it to fit the suggestion, although it is `reingest-data` instead to follow the instructions provided for the frontend.